force-app/main/educationCloudSettings/lwc/edcReleaseManagementCard/edcReleaseManagementCard.css

```diff
@@ -0,0 +1,30 @@
+.progress-container {
+    height: 30px;
+}
+.progressring-container {
+    max-width: 30px;
+}
+.progressring-content {
+    vertical-align: middle;
+}
+.notification-bell {
+    margin-top: -10px;
+    font-size: 15px;
+    display: block;
+    position: absolute;
```

Comment:
Consider replacing this with the `slds-is-absolute` class on the bell itself.
```suggestion
```

Reply:
Done, also replaced pixel sizes with relative units